### ISSUE TITLE
remove stop-loss and roi

### ISSUE THREAD

**Issue by suhail1142r** [freqtrade/freqtrade#5925]
## Describe your environment

  * Operating system: ____(Microsoft Windows [Version 10.0.19043.1348])
  * Python Version: _____ (`python - 3.10.0`)
  * Freqtrade Version: ____ (`freqtrade - 2021.10)
  
## Your question

*Ask the question you have not been able to find an answer in the [Documentation](https://www.freqtrade.io/en/latest/)*

**HELLO 
How can I remove stop-loss and ROI from the strategy without errors?**

>
 # Minimal ROI designed for the strategy.
    # This attribute will be overridden if the config file contains "minimal_roi".
    minimal_roi = {
        "60": 0.01,
        "30": 0.02,
        "0": 0.04
    }

    # Optimal stoploss designed for the strategy.
    # This attribute will be overridden if the config file contains "stoploss".
    stoploss = -0.10
>

**Comment by moinonin** [comment]
I assume you don't want to use those in your strategy. To do that, use unrealistically high values e.g.:
`minimal_roi = {
    "0": 1000
}`

do the same for stoploss:
`stoploss = -1000`

That said, what you are about to do is dangerous. I don't know why you would want to do that. Unless you want to just buy and hold or use a very good sell signal.

**Comment by xmatthias** [comment]
I'll assume this as answered by the above, thanks for this @moinonin! :100: 

Feel free to comment below / reopen if that's not the case.
